pyproject.toml

```diff
@@ -15,7 +15,7 @@ readme = "README.md"
 [tool.poetry.dependencies]
```

Comment:
I would probably include optional dependencies using pip install turfttopics[opentsne] otherwise you might get incompatible versions.

Reply:
will do